pkg/reconciler/v1alpha1/autoscaling/kpa/resources/service_test.go

```diff
@@ -22,12 +22,20 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 
 	pav1a1 "github.com/knative/serving/pkg/apis/autoscaling/v1alpha1"
+	"github.com/knative/serving/pkg/apis/networking"
 	"github.com/knative/serving/pkg/apis/serving"
+	"github.com/knative/serving/pkg/resources"
 	autoscalingv1 "k8s.io/api/autoscaling/v1"
 )
 
 var boolTrue = true
 
+func serviceLabels(labels map[string]string) map[string]string {
```

Comment:
I think this function is redundant, just add the missing key to the constant list we have.